internal/swiftoneofhelper/testdata/proto/simple_oneof.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package testdata;
+
+option go_package = "github.com/SebastienMelki/sebuf/internal/oneofhelper/testdata";
```

Comment:
careful here the go_package is wrong (oneofhelper vs switfoneofhelper). In any case we can change the style of .proto file to use relative pathing and not need to have this option line anyway. Will add it to the backlog